pkg/driver/options.go

```diff
@@ -87,6 +87,9 @@ type Options struct {
 	ReservedVolumeAttachments int
 	// ALPHA: WindowsHostProcess indicates whether the driver is running in a Windows privileged container
 	WindowsHostProcess bool
+	// LegacyXFSProgs enables the use of `` when formatting a volume with XFS. This is necessary if the driver is used
+	// to mount XFS volumes on Linux Nodes with a kernel version ≥ 5.4 due to __.
```

Comment:
This comment needs to be updated (please complete `use of '' ` and  `due to __` statements). Besides this, changes introduced in this PR lgtm.